README.md

```diff
@@ -98,13 +126,25 @@ class SimpleAgent(Agent):
         return action, new_state, 0.0
 ```
 
-Notice how it's pretty simple. We have to do some bookkeeping - namely appending messages as they come and passing tools. There is no magic here.
+An agent has two methods:
+
+```py
+agent_state = await agent.init_state(tools=tools)
+new_action, new_agent_state, value = await agent.get_asv(agent_state, obs)
+```
 
-### Compute Graph
+- The `get_asv(agent_state, obs)` method chooses an action (`a`) conditioned on the observation messages
+  returning the next agent state (`s`) and a value estimate (`v`).
+- The first argument, `agent_state`, is an optional container of memories which could include a list of previous actions and observations.
```

Comment:
The agent state is not specific to memories, but this sentence makes it seem that way. Can you rephrase to make memory-storage just be a possible use case

Reply:
Will do, what else could it contain?